Primo.js

```diff
@@ -274,6 +281,7 @@ function importPNX(text) {
 		if(creators[i]) {
 			var creator  = ZU.unescapeHTML(creators[i].textContent).split(/\s*;\s*/);
 			for(j in creator){
+				creator[j] = creator[j].replace(/\([\d\sabcefilr.? ]*[-–][\d\abcefilr.? ]*\)/gi, '');
```

Comment:
I have seen in your example that you want to replace for example "circa" and some date range. However, the regexp could potentially match some other stuff as well and I feel it would never be complete (if we think about different languages etc.). I could imagine to delete everything in parenthesis for the creator's name (and we should treat the contibutor then the same), but I haven't made my mind if that would be always a good idea...